python-sdk/tests/test_example_dags.py

```diff
@@ -47,47 +47,23 @@ def session():
     return get_session()
 
 
-@pytest.mark.parametrize(
-    "dag_id",
-    [
-        "example_amazon_s3_postgres",
-        "example_amazon_s3_postgres_load_and_save",
-        "example_amazon_s3_snowflake_transform",
-        "example_google_bigquery_gcs_load_and_save",
-        "example_snowflake_partial_table_with_append",
-        "example_sqlite_load_transform",
-        "example_append",
-        "example_load_file",
-        "example_transform",
-        "example_merge_bigquery",
-        "example_transform_file",
-        "calculate_popular_movies",
-    ],
+dag_bag = DagBag(Path(__file__).parent.parent / "example_dags")
+airflow_2_3 = pytest.mark.skipif(
+    airflow.__version__ < "2.3.0", reason="Require Airflow version >= 2.3.0"
 )
-def test_example_dag(session, dag_id):
-    dir_path = os.path.dirname(os.path.realpath(__file__))
-    db = DagBag(dir_path + "/../example_dags")
-    dag = db.get_dag(dag_id)
-
-    if dag is None:
-        raise NameError(f"The DAG with dag_id: {dag_id} was not found")
-    wrapper_run_dag(dag)
+airflow_2_3_dag_ids = {
+    "example_dynamic_map_task",
+    "example_dynamic_task_template",
+}
+dag_ids = [
+    pytest.param(dag_id, marks=airflow_2_3) if dag_id in airflow_2_3_dag_ids else dag_id
+    for dag_id in dag_bag.dag_ids
+]
```

Comment:
Following @kaxil 's feedback, perhaps we could have a simple test asserting `dag_ids` is not empty

Reply:
Done 0a6f104. :)